src/main/java/com/example/car_sharing_sertvice_project/model/User.java

```diff
@@ -10,27 +10,24 @@
 import jakarta.persistence.ManyToMany;
 import jakarta.persistence.Table;
 import java.util.Set;
-import lombok.EqualsAndHashCode;
 import lombok.Getter;
 import lombok.Setter;
 
 @Entity
 @Table(name = "users")
 @Getter
 @Setter
-@EqualsAndHashCode
 public class User {
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
-    private Long id;
+    private Integer id;
     @Column(unique = true)
     private String email;
     private String firstName;
     private String lastName;
     private String password;
-    @EqualsAndHashCode.Exclude
```

Comment:
Put EqualsAndHashCode.Exclude back together with EqualsAndHashCode as class annotation in all models